melodies_monet/driver.py

```diff
@@ -666,6 +666,8 @@ def sum_variables(self):
                         print('The variable name, {}, already exists and cannot be created with variable_summing.'.format(var_new))
                         raise ValueError
                     var_new_info = self.variable_summing[var_new]
+                    if self.variable_dict is None:
+                        self.variable_dict = {}
                     self.variable_dict[var_new] = var_new_info
```

Comment:
Can we add these same lines above to the obs class under def sum_variables too? This code looks identical so it should probably be listed in both places.

Reply:
I'll do this in a separate PR as discussed in meeting.